packages/packages/commonwell-cert-runner/src/document-consumption.ts

```diff
@@ -112,15 +106,27 @@ export async function documentConsumption(commonWell: CommonWell, queryMeta: Req
   if (!patientIdForDocQuery) {
     throw new Error(`[E1c] Could not determine patient ID for document query`);
   }
-  console.log(`... patientIdForDocQuery: ${patientIdForDocQuery}`);
 
   console.log(`>>> [E1c] Querying for docs...`);
   const respDocQuery = await commonWell.queryDocument(queryMeta, patientIdForDocQuery);
-  // console.log(JSON.stringify(respDocQuery, undefined, 2));
-  if (respDocQuery.entry.length > 0) {
-    const docs = respDocQuery.entry;
-    for (const doc of docs) {
-      console.log(`... Now would download file from URL: ${doc.content.location}`);
-    }
-  }
+
+  return respDocQuery.entry;
+}
+
+export async function retrieveDocument(
+  commonWell: CommonWell,
+  queryMeta: RequestMetadata,
+  doc: Document
+): Promise<void> {
+  // E2: Document Retrieve
+  console.log(`>>> E2c: Retrieve documents using FHIR (REST)`);
+
+  const fileName = `./commonwell_${nanoid()}.file`;
```

Comment:
Using `nanoid()` bc `id` and `identifier.value` return a string with `:`, and could potentially return invalid chars for filename.

We should store the file name on the DB at some point, so we can have any alphanumeric file names.